### PR TITLE
Temporarily offer a Timer-based blink controller to avoid excessive FPS (Bandaid #1253)

### PR DESCRIPTION
**Pull request by matthew-carroll**
Temporarily offer a Timer-based blink controller to avoid excessive FPS (Bandaid #1253)

Our hope is that the Flutter team solves https://github.com/flutter/flutter/issues/131357 to give us a singular approach that works in production and in tests.

### CODE REVIEW

**Review by brian-superlist — APPROVED** [review]
Looks like a reasonable bandaid to improve battery performance while avoiding refactoring thousands of tests.

File: super_text_layout/lib/src/infrastructure/blink_controller.dart [hunk 1]
```diff
@@ -20,19 +22,28 @@ class BlinkController with ChangeNotifier {
     _ticker = tickerProvider.createTicker(_onTick);
   }
 
+  BlinkController.withTimer({
+    Duration flashPeriod = const Duration(milliseconds: 500),
```
Comment (brian-superlist) [inline super_text_layout/lib/src/infrastructure/blink_controller.dart:26]: Not an important improvement to make here, but should this `flashPeriod` duration be configurable by the options passed to `DefaultCaretOverlayBuilder` as well?

Comment (matthew-carroll) [reply super_text_layout/lib/src/infrastructure/blink_controller.dart:26]: Probably. If anyone ever asks for it, we can add it. Although, so far, no one has.

**Comment by matthew-carroll** [comment]
@knopp would you like to review this before I merge, to ensure it solves your FPS problem?

**Review by knopp — APPROVED** [review]
Thanks!

LGTM!

**Comment by matthew-carroll** [comment]
Note: Apparently I didn't expose the necessary Timer controls for a SuperTextField. Add that API and then merge in.

**Review by angelosilvestre — COMMENTED** [review]
LGTM

**Pull request merged**